flake.nix

```diff
@@ -17,30 +20,36 @@
     "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
   ];
 
-  outputs = { self, nixpkgs, rust-overlay, flake-utils, ... }:
+  outputs = inputs: with inputs;
```

Comment:
It is more idiomatic to write this as: `outputs = inputs@{ nixpkgs, rust-overlay, ... }`